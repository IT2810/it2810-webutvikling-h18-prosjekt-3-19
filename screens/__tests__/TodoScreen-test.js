import React from "react";
import TodoScreen from "../TodoScreen";
import renderer from "react-test-renderer";

test("TodoScreen renders correctly", () => {
  const tree = renderer.create(<TodoScreen />);
  expect(tree.toJSON()).toMatchSnapshot();
});
