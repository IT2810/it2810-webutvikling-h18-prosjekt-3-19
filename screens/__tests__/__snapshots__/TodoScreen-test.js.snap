// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoScreen renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View>
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      placeholder="Write Todo Here!"
      placeholderTextColor="black"
      style={
        Object {
          "alignSelf": "stretch",
          "backgroundColor": "white",
          "borderTopColor": "#ededed",
          "borderTopWidth": 2,
          "color": "#000",
          "padding": 20,
        }
      }
      underlineColorAndroid="transparent"
      value=""
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#E91E63",
          "borderRadius": 5,
          "height": 40,
          "justifyContent": "center",
          "opacity": 1,
          "position": "absolute",
          "right": 10,
          "top": 10,
          "width": 40,
          "zIndex": 11,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#fff",
            "fontSize": 24,
          }
        }
      >
        +
      </Text>
    </View>
  </View>
</View>
`;
